packages/components/src/form/inputs/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,93 @@
+import { cns } from '@stcland/utils'
+import { useFormContext } from 'react-hook-form'
+import { nullFormContext } from '../Form'
+import FormLabel from '../FormLabel'
+
+//*****************************************************************************
+// Interface
+//*****************************************************************************
+type RadioButtonShape = {
+  label: string;
+  value: string;
+};
+interface Props {
+  name: string;
+  label?: string;
+  direction?: 'vertical' | 'horizontal';
+  defaultChecked?: string;
+  radioButtons: RadioButtonShape[];
+  readOnly?: boolean;
+  required: boolean;
```

Comment:
Why is `required` the only required prop(LOL)? if we are setting a default value?

Reply:
radioButtons (the options) is also required so is name